cloud/amazon/cloudformation.py

```diff
@@ -143,6 +148,21 @@
       ClusterSize: 3
     tags:
       Stack: ansible-cloudformation
+
+# Use a template with a lookup
+- name: launch ansible cloudformation example
+  cloudformation:
+    stack_name="ansible-cloudformation" state=present
+    region=us-east-1 disable_rollback=true
+    template_contents="{{ lookup('file', 'cloudformation.json') }}"
```

Comment:
This example would actually fail because the JSON file contents would be parsed into a Python type.